docs/src/config/stepconf.adoc

```diff
@@ -98,63 +98,21 @@ The LinuxCNC Configuration Selector has configs for Sherline already configured.
   Enter the result of the Latency Test here.
   To run a latency test press the 'Test Base Period Jitter' button. See the
   <<sec:latency-test,Latency Test>> section for more details.
+
+.Latency Test
+image::images/latency-test_en.png["Latency Test",align="center"]
+
 * 'Max Step Rate' -(((Max Step Rate)))
   StepConf automatically calculates the Max Step Rate based
   on the driver characteristics entered and the latency test result.
 * 'Min Base Period' - (((Min Base Period)))
   StepConf automatically determines the Min Base Period
   based on the driver characteristics entered and latency test result.
 
-[[sub:latency-test]]
-== Latency Test(((Latency Test)))
-
-While the test is running, you should 'abuse' the computer. Move
-windows around on the screen. Surf the web. Copy some large files
-around on the disk. Play some music. Run an OpenGL program such as
-glxgears. The idea is to put the PC through its paces while the latency
-test checks to see what the worst case numbers are.  Run the test at least a few
-minutes. The longer you run the test the
-better it will be at catching events that might occur at less frequent
-intervals. This is a test for your computer only, so no hardware needs
-to be connected to run the test.
-
-[WARNING]
-Do not attempt run LinuxCNC while the latency test is running.
-
-.Latency Test
-image::images/latency-test_en.png["Latency Test",align="center"]
-
-Latency is how long it takes the PC to stop what it is doing and
-respond to an external request. In our case, the request is the
-periodic 'heartbeat' that serves as a timing reference for the step
-pulses. The lower the latency, the faster you can run the heartbeat,
-and the faster and smoother the step pulses will be.
-
-Latency is far more important than CPU speed. The CPU isn't the only
-factor in determining latency. Motherboards, video cards, USB ports,
-SMI issues, and a number of other things can hurt the latency.
-
-.Troubleshooting SMI Issues (LinuxCNC.org Wiki)
-************************************************************
-Fixing Realtime problems caused by SMI on Ubuntu
-
-https://wiki.linuxcnc.org/cgi-bin/wiki.pl?FixingSMIIssues
-************************************************************
 
-The important numbers are the 'max jitter'. In the example above 9075
+The important number from the result of the Latency Test is the 'max jitter'. In the example above, 9075
 nanoseconds (ns), or 9.075 microseconds (µs), is the highest jitter.
-Record this number, and enter it in the Base Period Maximum Jitter box.
-
```

Comment:
Again this is good information except it is referencing software stepping requirements.